recipes/naif_earth_itrf93/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "naif_earth_itrf93" %}
+{% set version = "2007.4.3.1" %}
+{% set python_min = "3.9" %}
```

Comment:
We only need to set `python_min` if the min version is higher than conda-forge's min. 

```suggestion
```